c7n/resources/appmesh.py

```diff
@@ -174,23 +212,42 @@ class resource_type(TypeInfo):
         service = 'appmesh'
 
         # https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsappmesh.html#awsappmesh-resources-for-iam-policies  # noqa
+        # TODO IS THIS EVEN USED AT RUNTIME?
+        # Junk value doesn't seem to affect the functionality of the extension
+        # but does fail the test PolicyMetaLint.test_valid_arn_type
+        # but that isn't a functional test.
         arn_type = "mesh"
 
         # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appmesh-virtualgateway.html  # noqa
-        cfn_type = config_type = 'AWS::AppMesh::VirtualGateway'
+        # TODO IS THIS EVEN USED AT RUNTIME?
+        # Junk value doesn't seem to affect the functionality of the extension
+        # but does fail the test PolicyMetaLint.test_cfn_resource_validity
+        # but that isn't a functional test.
+        cfn_type = 'AWS::AppMesh::VirtualGateway'
+
+        # TODO IS THIS EVEN USED AT RUNTIME?
+        # Junk value doesn't seem to affect the functionality of the extension
+        # but does fail the test PolicyMetaLint.test_cfn_resource_validity
+        # but that isn't a functional test.
+        config_type = 'AWS::AppMesh::VirtualGateway'
 
         # if a resource type is supported by resource group tagging
         # api setting this value get tag filters/actions
         universal_taggable = object()
 
-        # id: Path to "id" field in the
+        # id: Path to "id" field in the .... IN THE WHAT???
+        # TODO: WHERE IS THIS EVEN USED AT RUNTIME?
         id = 'meshName'
 
         # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appmesh-virtualgateway.html  # noqa
-        arn = "metadata.arn"
+        # TODO: WHERE IS THIS EVEN USED AT RUNTIME?
+        arn = "metadata.arn!!!!!!!"
+
+        # TODO: WHERE IS THIS EVEN USED AT RUNTIME?
+        name = 'virtualGatewayName!!!!!!!!!!'
 
-        name = 'virtualGatewayName'
-        date = 'createdAt'
+        # TODO: WHERE IS THIS EVEN USED AT RUNTIME?
+        date = 'createdAt!!!!!!!!!'

```

Comment:
Defining this will [add it to the list of default reporting output fields](https://github.com/cloud-custodian/cloud-custodian/blob/503d73d89d3b97e94d97b2c8bdc164d0acac40e8/c7n/reports/csvout.py#L163-L164)

Reply:
Hi where can I see these fields output?

Reply:
using `custodian report`

Reply:
Hi I'm sorry I did try to figure out how to run the report but wasn't sure what I was doing. Are you able to help. Apols

Reply:
Figured out a test